binderhub/health.py

```diff
@@ -119,15 +119,19 @@ def initialize(self, hub_url=None):
         return True
 
     @false_if_raises
+    @at_most_every(interval=10)
     @retry
     async def check_docker_registry(self):
         """Check docker registry health"""
+        app_log.info("Checkinng registry status")
```

Comment:
```suggestion
        app_log.info("Checking registry status")
```

Reply:
I need to take my laptop to the Apple store to get the keyboard fixed. Even the software fix to stop repeated keys ain't working well enough any more :-/